tools/replay/ReplayFile.hpp

```diff
@@ -99,29 +104,45 @@ class ReplayFile {
     } argv ;
   };
 
+  const uint64_t REPLAY_MAGIC =
```

Comment:
Magic numbers?! 😄

Reply:
Yup.  Used to confirm that the file is an actual binary created by the replay tool.